openassessment/xblock/submission_mixin.py

```diff
@@ -64,6 +53,10 @@ class SubmissionMixin(object):
         'htm', 'html',
     ]
 
+    @cached_property
```

Comment:
I am curious, what is the reason behind the use of cached_property ?

Reply:
"The @cached_property decorator caches the result of a method with a single self argument as a property. The cached result will persist as long as the instance does, so if the instance is passed around and the function subsequently invoked, the cached result will be returned."  So it's an easy way of not having a new `FileUploadManager` instantiated each time `self.upload_manager` is referenced.  Not a big deal here, because `FileUploadManager` doesn't store much state.